src/components/DisplayBooks.js

```diff
@@ -12,23 +14,34 @@ const DisplayBook = (props) => {
   };
 
   return (
-    <div>
-      <ul>
+      <ul className={styles.booksContainer}>
         {bookList.map((book) => (
-          <li key={book.item_id}>
-            <p>
-              Title:
-              { book.title }
-            </p>
-            <p>
-              Author:
-              { book.category }
-            </p>
-            <button type="button" onClick={() => { removeBookStore(book.item_id); }}>Remove Book</button>
+          <li key={book.item_id}
+          className={styles.liCard}>
+            <div className={styles.bookCard}>
+              <p className={styles.title}>
+                { book.title }
+              </p>
+              <p className={styles.category}>
```

Comment:
- [ ] The book category should be styled as shown in the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/132089241-86921327-b144-42bd-8663-4501af7ac780.png)